server.js

```diff
@@ -287,7 +287,7 @@ function getMockConfigData() {
         },
         {
           spec: {
-            clusterServiceClassExternalName: 'amq-broker-72-persistence'
+            clusterServiceClassExternalName: 'amq-online-standard'
```

Comment:
I was expecting all removals. Why do we point at a different AMQ?

Reply:
AMQ has been replaced with AMQ Online (which as far as i understand comes with the cluster by default).

Reply:
:+1: